openssl-sys/src/evp.rs

```diff
@@ -20,7 +20,7 @@ pub const EVP_PKEY_X448: c_int = NID_X448;
 pub const EVP_PKEY_ED448: c_int = NID_ED448;
 pub const EVP_PKEY_HMAC: c_int = NID_hmac;
 pub const EVP_PKEY_CMAC: c_int = NID_cmac;
-#[cfg(ossl110)]
+#[cfg(any(ossl110, boringssl))]
```

Comment:
boringssl doesn't use the bindings in openssl-sys.

Reply:
reverted the changes in openssl-sys